src/Site_WP_Command.php

```diff
@@ -254,7 +254,6 @@ public function info( $args ) {
 		$info   = [
 			[ 'Site', $prefix . $this->site['name'] ],
 			[ 'Access phpMyAdmin', $prefix . $this->site['name'] . '/ee-admin/pma/' ],
-			[ 'Access mailhog', $prefix . $this->site['name'] . '/ee-admin/mailhog/' ],
```

Comment:
Can you change the order? Putting the Wordpress user and Wordpress pass at the top? Then the DB details. Then the admin tools info.